src/Microsoft.Framework.PackageManager/NuGet/Core/Configuration/Settings.cs

```diff
@@ -172,15 +169,11 @@ public string ConfigFilePath
             if (configFileName == null)
             {
                 // load %AppData%\NuGet\NuGet.config
-#if DNX451
-                var appDataPath = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
-#else
-                var appDataPath = Environment.GetEnvironmentVariable("APPDATA");
-#endif
-                if (!String.IsNullOrEmpty(appDataPath))
+                var userSettingsDir = DnuEnvironment.GetFolderPath(DnuFolderPath.UserSettingsDirectory);
```

Comment:
It is `AppData/NuGet`, instead of `AppData/NuGet/Config` because the comments say so